apps/web/app/components/hero/hero.css

```
@@ -0,0 +1,348 @@
+/* ============================================================
+   AEGISCODE HERO — Revised Design
+   ============================================================ */
+
+/* ---------- Keyframes ---------- */
+@keyframes cursorBlink {
```

Stylelint (17.3.0)

[error] Expected keyframe name "cursorBlink" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)